tests/issue-442-build-config-name/src/test/java/io/dekorate/examples/kuberentes/Issue442BuildConfigNameTest.java

```diff
@@ -35,7 +35,7 @@ public void shouldHaveCustomBuildConfigNameYaml() {
     assertNotNull(list);
     BuildConfig b = findFirst(list, BuildConfig.class).orElseThrow(() -> new IllegalStateException());
     assertNotNull(b);
-    assertEquals("o-name", b.getMetadata().getName());
+    assertEquals("issue-442-build-config-name", b.getMetadata().getName());
```

Comment:
I'm not really sure I understand where this is coming from, can you please elaborate?

Reply:
When we use a custom name for Openshift resources it should only affect things like: 
- DeploymentConfig
- Service
- Route

It should **NOT** affect BuildConfig and ImageStreams. Why?
Because all things s2i are customized independently.

This is what this test case is about.

Reply:
Great, thanks for the detailed explanation